.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
I... absolutely hate that. Why the hell is there comments in a JSON file? It's autogenerated by VSCode everywhere I've seen and it loads fine so whatever but it's still gross.

Reply:
Didn't even mean to add that, putting .vscode in the ignore